app/views/chatroom/index.html.erb

```diff
@@ -1,63 +1,85 @@
                 <div class="summary">
-                  <a>Christian Rocha</a> signed up for the site.
+                  <div class="checbox-container">
+                    <input type="checkbox" , value=<%= user.id %> , class="test"></div>
+                  <div class="user-container">
+                    <h4><%= user.name.titleize %></h4>
+                    <p><%= user.email %></p>
+                  </div>
                 </div>
               </div>
             </div>
           </div>
         </div>
-      </div>
-      <div class="extra content">
-        <form class="ui reply form">
-          <div class="field">
-            <div class="ui fluid icon input">
-              <input type="text" placeholder="Enter Your Message . . . ." >
-              <i class="bordered inverted orange edit icon"></i>
-            </div>
-          </div>
-        </form>
+      <% end %>
+      <div class="group-create-container">
+        <%= text_field_tag 'name', nil, placeholder: 'Enter Group Name', class: 'input-resize' %>
+        <%= link_to "Create", "", class: "ui orange submit button create-group" %>
       </div>
     </div>
-    <div class="four wide column">
-          <div class="ui inverted vertical menu">
-            <a class="active item">
-              Online
-            </a>
-            <a class="item">
-              Messages
-            </a>
-            <a class="item">
-              Friends
-            </a>
-          </div>
-       </div>
-  </div>
\ No newline at end of file
+  </div>
+  <div>
+    <table class="ui celled striped table">
+      <thead>
+      <tr>
+        <th colspan="3">
+          Groups
+        </th>
+      </tr>
+      </thead>
+      <tbody>
+      <% @chatroom.each do |element| %>
+        <tr>
+          <td class="collapsing">
+            <i class="folder icon"></i>
+            <%= link_to element.description, new_chatroom_path(element) %>
+          </td>
+          <td><%= element.users.all.count %></td>
+          <td><%= link_to  chatroom_path(element), method: :delete, data: { confirm: 'Are you sure?' } do %>
+              <i class="fa fa-trash"></i>
+            <% end %>
+          </td>
+        </tr>
+      <% end %>
+      </tbody>
+    </table>
+  </div>
+</div>
+
+<script>
+    var members = [];
+    $(".test").change(function () {
+        if (members.includes(this.value)) {
+            members.pop(this.value)
+        } else {
+            members.push(this.value)
+        }
+    });
+    $('.create-group').click(function () {
+        var title = $(this).siblings('input').val()
+        $.ajax({
+            url: "/chatroom",
+            type: "post",
+            data: {
+                group_members: members,
+                group_title: title
+            },
+            dataType: "json",
+            success: function (response) {
+            },
+            error: function (response) {
+            }
+        })
+    });
+</script>
```

Comment:
Use stimulus controller properly for this JS, we dont prefer to use JS in Html views, need to separate them